packages/launchpad/src/test/launchpad.tsx

```diff
@@ -1,9 +1,13 @@
-import { render, screen } from '@testing-library/react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
-import LaunchpadInternal from '../launchpad-internal';
+import LaunchpadInternal, { LaunchpadInternalProps } from '../launchpad-internal';
```

Comment:
Nit: this should be a type import.
```suggestion
import LaunchpadInternal, { type LaunchpadInternalProps } from '../launchpad-internal';
```

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/87141/commits/322631506ef06de292a7c037760c8c647804fc8b